db/blob/blob_file_builder_test.cc

```diff
@@ -145,7 +145,7 @@ TEST_F(BlobFileBuilderTest, BuildAndCheckOneFile) {
       TestFileNumberGenerator(), fs_, &immutable_options, &mutable_cf_options,
       &file_options_, job_id, column_family_id, column_family_name, io_priority,
       write_hint, nullptr /*IOTracer*/, nullptr /*BlobFileCompletionCallback*/,
-      &blob_file_paths, &blob_file_additions);
+      options.cf_paths.front().path, &blob_file_paths, &blob_file_additions);
```

Comment:
There is another PR in the works to introduce a "blob_db_path" option.  To make things simpler in the future, can we introduce a method now to ImmutableCFOptions "GetBlobDBPath() which returns cf_paths.front()?  

Tracking down all of the places that use cf_paths.front to mean blob_db_path is not simple.

Reply:
Seems to me this is actually the new `dbname` argument? If so, we could probably pass a hardcoded value in these tests.